### PR TITLE
Documentation,charts,manifests: Update the downstream logo.

### PR DESCRIPTION
**Pull request by timflannagan1**
This updates the various values.yaml helm charts that are used to generate the YAML manifests.

In order to generate this icon binary, I ran the following command:

```bash
$ base64 -i -w 0 Documentation/downstream_metering_logo.svg
```

And I verified that was working properly but running the following:

```bash
$ output=$(base64 -i -w 0 Documentation/downstream_metering_logo.svg)
$ echo $output | base64 -d | display
```

Note(s):
- Added a csv.icon configuration for the upstream-values.yaml to distinguish between the upstream and downstream logos, as before they pointed to the same logo.
- While testing, I rename the package.yaml we use to avoid conflicting with the metering-ocp packagemanifest that gets loaded in the redhat-operators CatalogSource. It looks like the console attempts to load the first metering-ocp packagemanifest that gets returned when I was poking around chrome's element inspection view.

Screenshot from 2020-07-20 17-46-54

### CODE REVIEW

**Comment by timflannagan1** [comment]
Let's hold off on merging this until Sergio is back on PTO.
/hold

File: charts/metering-ansible-operator/upstream-values.yaml [hunk 1]
```diff
@@ -80,6 +80,10 @@ olm:
 
       * **Configure AWS Billing Data Source** - To assign Pod $$ costs on AWS, create a [read-only IAM user](https://github.com/kube-reporting/metering-operator/blob/master/Documentation/configuring-aws-billing.md) ([example-policy](https://github.com/kube-reporting/metering-operator/blob/master/Documentation/aws/read-only.json)) and [configure Metering](https://github.com/kube-reporting/metering-operator/blob/master/manifests/metering-config/aws-billing.yaml) to use it.
 
+    icon:
```
Comment (bentito) [inline charts/metering-ansible-operator/upstream-values.yaml:83]: We should put the commit message into perhaps its own `dev-notes.txt` doc in https://github.com/kube-reporting/setup perhaps? Or somewhere devs can find it to be able to repeat this rarely done technique.

Comment (timflannagan1) [reply charts/metering-ansible-operator/upstream-values.yaml:83]: Yeah that's not a bad idea, but it should be sufficient for now to just refer back to the PR description/commit message for this and emulate the steps listed there.

**Pull request merged**